src/html/userSurvey.popupPrompt.html

```diff
@@ -41,26 +46,53 @@
             }
 
             .buttonGray:hover {
-                background-color: #b5b5b5;
+                background-color: #e2e2e2;
+            }
+
+            .icon {
+                max-width: 50px;
+                max-height: 50px;
+                float: left;
+                padding-right: 10px;
+            }
+
+            .separator {
+                width: 1px;
+                height: 46px;
+                color: black;
+                float: left;
+                background: #d3d3d3;
             }
         </style>
     </head>
     <body>
-        <div id="content" class="paragraph" style="width: 300px; padding-right: 25px; padding-left: 25px;">
+        <div style="display: flex; max-width: 280px; padding: 10px;">
+            <img id="icon" class="icon" style="display: none;">
+            <div id="content" class="paragraph" style="padding-left: 10px; padding-right: 10px;"></div>
         </div>
         <div class="button-container">
-            <a href="#" class="buttonGray" name="later">Later</a>
-            <a href="#" class="buttonGray" name="never">Never</a>
-            <a href="#" class="buttonBlue" name="agree">Continue</a>
+            <a href="#" class="button buttonGray" name="later">Later</a>
+            <span class="separator"></span>
+            <a href="#" class="button buttonGray" name="never">Never</a>
+            <a href="#" class="button buttonBlue" name="agree">Continue</a>
         </div>
         <script>
             (async function() {
                 // Load the message from storage
-                const storageKey = "webScience.userSurvey.popupPromptMessage";
-                const contentResults = await browser.storage.local.get(storageKey);
-                if(storageKey in contentResults) {
+                const messageStorageKey = "webScience.userSurvey.popupPromptMessage";
+                const messageResults = await browser.storage.local.get(messageStorageKey);
+                if(messageResults[messageStorageKey]) {
                     const contentElement = document.getElementById("content");
-                    contentElement.innerHTML = contentResults[storageKey];
+                    contentElement.innerHTML = messageResults[messageStorageKey];
```

Comment:
`innerHTML` might be OK, but it may lead to additional scrutiny. `createElement` et al. might be better.

Reply:
How about `innerText`?

Reply:
Yes! If all you need to modify is text, then this is a great way to [prevent XSS attacks](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#differences_from_innerhtml) in case unsanitized user input is ever accidentally passed here.

Reply:
Great, I switched this to use `textContent`.